sound/soc/sof/topology.c

```diff
@@ -2240,8 +2240,7 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	}
 
 	ipc_size = sizeof(struct sof_ipc_comp_process) +
-		le32_to_cpu(private->size) +
-		ipc_data_size;
+		ipc_data_size + sizeof(swidget->comp_ext);
```

Comment:
why remove 'le32_to_cpu(private->size) ' ?

Reply:
@plbossart I think this was an error in previous code, maybe we should make a separated PR to fix this first @marc-hb .
Please see the discussion about this here https://github.com/thesofproject/linux/pull/2090#discussion_r442689518